hack/update-generated-protobuf.sh

```diff
@@ -13,24 +13,10 @@ if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc 3.0."* ]]; the
   exit 1
 fi
 
-if [[ -z "$(which goimports)" ]]; then
-  echo "goimports is required to be present on your path in order to format the generated"
-  echo "protobuf files"
-  echo
-  exit 1
-fi
-
+os::util::ensure::system_binary_exists 'goimports'
 os::build::setup_env
 
-"${OS_ROOT}/hack/build-go.sh" tools/genprotobuf vendor/k8s.io/kubernetes/cmd/libs/go2idl/go-to-protobuf/protoc-gen-gogo
-genprotobuf="$( os::build::find-binary genprotobuf )"
-
-if [[ -z "${genprotobuf}" ]]; then
-	echo "It looks as if you don't have a compiled genprotobuf binary."
-	echo
-	echo "If you are running from a clone of the git repo, please run"
-	echo "'./hack/build-go.sh tools/genprotobuf'."
-	exit 1
-fi
+os::util::ensure::built_binary_exists 'genprotobuf'
+os::util::ensure::built_binary_exists 'protoc-gen-gogo' vendor/k8s.io/kubernetes/cmd/libs/go2idl/go-to-protobuf/protoc-gen-gogo
 
-PATH="$( dirname "${genprotobuf}" ):${PATH}" ${genprotobuf} --output-base="${GOPATH}/src" "$@"
+PATH="$( dirname "${genprotobuf}" ):${PATH}" genprotobuf --output-base="${GOPATH}/src" "$@"
```

Comment:
Is `PATH="$( dirname "${genprotobuf}" ):${PATH}" ` correct? We no longer set `${genprotobuf}`, and it looks like `genprotobuf` is built into the output directory, which is in `PATH`.